### DISCUSSION TITLE
Chart components  - Skeleton loader and color tokens

### DISCUSSION THREAD

**Discussion by ishah-twilio** [twilio-labs/paste#1710] [General] [answered]
Chart components  

Team / Product / Feature: IoT > Super SIM > Data insights
Background: Team is updating existing charts as well as adding new chart components specific to Super SIM Data insights features using Highchart. 

Looking for guidance on the 
1. Skeleton loading for (Pie chart, bar graph and map). Is pattern available for chart components skeleton loading? 
2. Color tokens and guidance to customize the Highchart 

<img width="1696" alt="charting components color and skeleton loader" src="https://user-images.githubusercontent.com/69016180/127539009-1d618dcd-2ef1-4464-b140-feea91d75dc4.png">

Thank you.

**Comment by SiTaggart** [comment, marked as answer]
Hi @ishah-twilio 

For Skeleton loading, we don't have any prescribed treatments as yet. The [skeleton loader](https://paste.twilio.design/components/skeleton-loader/) comes in a bunch of shapes and sizes, so it's pretty much able to do anything you want. You might be the first team to use it, so we're excited to see what you come up with.

As for Charting colors, we still haven't been able to get this prioritized but after a quick search of previous discussions, it does look like @kylegilbreath is working on some and might be contributing them to the system. Maybe sync with him and see where he's at. Perhaps you can work on it together.
